cookbooks/stateofthemap/templates/default/apache.jekyll.erb

```diff
@@ -38,14 +40,9 @@
         SSLCertificateFile /etc/ssl/certs/<%= @year %>.stateofthemap.org.pem
         SSLCertificateKeyFile /etc/ssl/private/<%= @year %>.stateofthemap.org.key
 
-        DocumentRoot /srv/<%= @year %>.stateofthemap.org/_site
+        RequestHeader set X-Forwarded-Proto “https”
+        RequestHeader set X-Forwarded-Port “443”
```

Comment:
Why are these needed?

Reply:
While this isn't used by the sotm or welcome containers, it is used to let the container backend know that the request is an HTTPS request. It might be desired in future for a backend to redirect / reject the request if it is not an HTTPS request.